src/main/java/org/folio/dew/service/BulkEditStatisticService.java

```diff
@@ -1,23 +1,26 @@
 package org.folio.dew.service;
 
 import org.springframework.batch.core.configuration.annotation.JobScope;
+import org.springframework.batch.core.configuration.annotation.StepScope;
 import org.springframework.stereotype.Service;
 
-@JobScope
+import java.util.concurrent.atomic.AtomicInteger;
+
+//@StepScope // remove
 @Service
 public class BulkEditStatisticService {
```

Comment:
Hi @obozhko-folio the scope should be specify here as now is singleton and could affect statistic for parallels execution.